integration_tests/suite/base/test_user_import.py

```diff
@@ -361,6 +361,26 @@ def test_given_csv_has_all_sip_fields_then_sip_endpoint_created():
     assert_that(sip, has_entries(username="sipusername", secret="sipsecret"))
 
 
+def test_given_csv_has_minimal_webrtc_fields_then_sip_endpoint_created():
+    csv = [
+        {"firstname": "Chârles", "line_protocol": "webrtc", "context": config.CONTEXT}
+    ]
+
+    response = client.post("/users/import", csv)
+    sip_id = get_import_field(response, 'sip_id')
+    line_id = get_import_field(response, 'line_id')
+
+    sip = confd.endpoints.sip(sip_id).get().item
+    assert_that(
+        sip,
+        has_entries(
+            id=sip_id,
+            line=has_entries(id=line_id),
+            options=has_items(contains('webrtc', 'yes')),
```

Comment:
Wouldn't it be best to test for all options required for WebRTC?

Reply:
Fixed